tests/multi_ptr/multi_ptr_arithmetic_op.h

```diff
@@ -0,0 +1,350 @@
+
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides code for multi_ptr arithmetic operators
+//
+*******************************************************************************/
+
+#ifndef __SYCLCTS_TESTS_MULTI_PTR_ARITHMETIC_OP_H
+#define __SYCLCTS_TESTS_MULTI_PTR_ARITHMETIC_OP_H
+
+#include "../common/common.h"
+
+#include "../common/section_name_builder.h"
+#include "multi_ptr_common.h"
+
+namespace multi_ptr_prefetch_member {
+
+constexpr int expected_val = 42;
+
+/**
+ * @brief Provides functor for verification on multi_ptr arithmetic operators
+ * @tparam T Current data type
+ * @tparam AddrSpaceT sycl::access::address_space enumeration's field
+ * @tparam IsDecoratedT sycl::access::decorated enumeration's field
+ */
+template <typename T, typename AddrSpaceT, typename IsDecoratedT>
+class run_multi_ptr_arithmetic_op_test {
+  static constexpr sycl::access::address_space space = AddrSpaceT::value;
+  static constexpr sycl::access::decorated decorated = IsDecoratedT::value;
+  using multi_ptr_t = sycl::multi_ptr<T, space, decorated>;
+
+ public:
+  /**
+   * @param type_name Current data type string representation
+   * @param address_space_name Current sycl::access::address_space string
+   *        representation
+   * @param is_decorated_name Current sycl::access::decorated string
+   *        representation
+   */
+  void operator()(const std::string &type_name,
+                  const std::string &address_space_name,
+                  const std::string &is_decorated_name) {
+    constexpr size_t array_size = 10;
+    // Array that will be used in multi_ptr
+    T arr[array_size];
+    for (size_t i = 0; i < array_size; ++i) {
+      arr[i] = i;
+    }
+    size_t middle_elem_index = array_size / 2;
+    T op_return_result_expected_val;
+    T op_calling_expected_val;
+
+    auto queue = sycl_cts::util::get_cts_object::queue();
+    T value = user_def_types::get_init_value_helper<T>(expected_val);
```

Comment:
I can't find where we use this variable

Reply:
This variable has been deleted [here](https://github.com/KhronosGroup/SYCL-CTS/pull/353/commits/e27b254c3dc043795b522cab15eed3133e8c6b47#diff-020505eb059da331d313f3451593a120219c86a4cb86846e1c5080d4f684d768L56)